op-e2e/actions/helpers/l2_sequencer.go

```diff
@@ -204,6 +204,13 @@ func (s *L2Sequencer) ActBuildL2ToTime(t Testing, target uint64) {
 	}
 }
 
+func (s *L2Sequencer) ActBuildL2ToCanyon(t Testing) {
```

Comment:
This seems like an unnecessary diff given all production networks today have canyon at genesis